.github/workflows/CI.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     name: OTP ${{matrix.otp}}
     strategy:
       matrix:
-        otp: ['22.3.4', '21.3']
+        otp: ['22.3.4', '23.1']
```

Comment:
This looks so much like an accident, needs a comment in the commit message. Why is this changed twice, in two different commits?

Reply:
It's not an accident but rather an iterative process:
* First I fixed testing as it was on Travis (with OTP 23.1).
* Then I did update of the dependencies
* Then i made it building and testing with OTP 23.1

Before `rebar.lock` update, it was not building on OTP 23.1 at all.

Reply:
Ok I see, it's just coincidence: `21.3` -> `23.1` look suspicious ;-)